resync/settings_validators.py

```
@@ -333,31 +385,77 @@ def validate_cross_field_consistency(self):
-            for sub_field in ("tws_timeout_connect", "tws_timeout_read",
-                              "tws_timeout_write", "tws_timeout_pool"):
-                sub_val = getattr(self, sub_field, None)
-                if sub_val is not None and sub_val > tws_request_timeout:
-                    errors.append(
-                        f"{sub_field} ({sub_val}s) > tws_request_timeout ({tws_request_timeout}s)"
-                    )
-
-        # 4. Production-specific cross-checks
-        env = getattr(self, "environment", None)
-        if env and str(env).lower() in ("production", "prod"):
+        tws_request_timeout = getattr(self, "tws_request_timeout")
```

Ruff (0.15.0)

[warning] Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)